ee/tabby-webserver/src/hub/mod.rs

```diff
@@ -133,4 +136,8 @@ impl Hub for Arc<HubImpl> {
             }
         }
     }
+
+    async fn get_repositories(self, _context: tarpc::context::Context) -> RepositoryCache {
+        META.clone()
```

Comment:
instead of reading from META, we should really reading this from sqlite with [RepositoryService](https://github.com/TabbyML/tabby/blob/84f27ce67d2ec63cdc9e476e988d54fdf4ebe65d/ee/tabby-webserver/src/schema/repository.rs#L33)